lib/util/clone.ts

```diff
@@ -1,5 +1,6 @@
 import { quickStringify } from './stringify';
 
 export function clone<T>(input: T | null = null): T {
-  return JSON.parse(quickStringify(input));
+  const sortedinput = quickStringify(input);
```

Comment:
is this really sorted? i thought it's only stringified  JSON 

we should add a test that this doesn't change key order.

Reply:
"sorted" is misleading actually my bad. It preseves the insertion order only.Test on the way👍

Reply:
rename to `stringifiedInput`?